src/shell.cpp

```diff
@@ -0,0 +1,107 @@
+#include "shell.h"
+
+Shell::Shell() {}
```

Comment:
Putting this here is redundant, you can just use `Shell() = default` in the header file